libraries/azurerm_virtual_machine_disk.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+# TODO: we will have a name collision with the existing implementation
```

Comment:
Could we reword this as something we can do?

Reply:
https://github.com/inspec/inspec-azure/pull/63#discussion_r199281985